test/JDBC/expected/securityadmin_role-vu-verify.out

```diff
@@ -0,0 +1,1268 @@
+-- tsql
```

Comment:
We should add test cases for following scenarios - 
1. Test cases in conjunction with GRANT ON SCHEMA support 
2. add test cases where user is member of both `sysadmin` and `securityadmin`, it should be able to connect to newly created database
3. we should also add negative tests cases where we try to add built in users to `securityadmin`
4. `create login securityadmin with password = '123'` should be error'd out
5. `alter server role securityadmin add member <db-role>` -- this should be error'd out also
6.

Reply:
Added.